### PR TITLE
Work with ActiveJob

### PR DESCRIPTION
**Pull request by misoobu**
Thanks for maintaining the awesome gem.

I am using Sidekiq with ActiveJob in my project.
This PR makes Sidecloq to work with ActiveJob, like sidekiq-scheduler.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,6 +22,12 @@ if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2")
   gem 'rack', '< 2.0'
 end
 
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.2.2")
```
Comment (mattyr) [inline Gemfile]: can we roll this into the conditional above? and maybe a comment for the reason why it's there

File: Rakefile [hunk 2]
```diff
@@ -35,6 +35,9 @@ task :web do
     config[:schedule] = sched
   end
 
+  class DoWork
```
Comment (mattyr) [inline Rakefile:38]: This shouldn't be here... right?

**Review by mattyr — COMMENTED** [review]
Awesome PR, thanks -- just a few minor changes

File: Gemfile (hunk 1 above)
Comment (misoobu) [reply Gemfile]: Certainly. I will fix this. 😃

File: Rakefile (hunk 2 above)
Comment (misoobu) [reply Rakefile:38]: When job is enqueued, Sidecloq needs existence class for determination that it is a kind of active_job or not. So I added this lines for prevent error that happen at web "Enqueue Now" action. What kind of solution should I fix?

Comment (mattyr) [reply Rakefile:38]: Ah right, ok this is fine.

**Comment by misoobu** [comment]
Thank you. 😄

**Pull request merged**